### ISSUE TITLE
Add Staff to Vendor P1

### ISSUE THREAD

**Issue by gwynnemeeks** [nss-evening-cohort-12/nutshell-muntjac#92]
### **User Story**
As a user, I should be able to assign a Staff member to a Vendor (#77)
---
As a user, when signed in I should be able to visit the staff member page and see a list of staff members. There should be a button that allows adding a staff member to a vendor. When the button is clicked a form should appear with radio buttons for each Staff Member.

### **AC**
---
**When** the user signs in and visits the vendors page they should see a list of vendors and a button that allows adding staff information to vendor
**When** the user clicks the add to vendor button a form with several radio buttons should populate the dom

### **To Do**
--- 

- [x] Create a new component that is name addStaffVendor
- [x] Create a new file named addStaffVendor
- [x] Create a function that builds a form and will print it to the dom.
- [x] Create multiple radio buttons on the form one for each vendor
- [x] Must be complete before starting issue #104

### **Moq Up**
---
![image](https://user-images.githubusercontent.com/63276406/88348543-54dc0700-cd13-11ea-9d60-e8f657038ce9.png)

**Comment by luketlancaster** [comment]
I might split this up...one card for adding staff to vendor, another for display staff on vendor. Same with rides. Also do you have any cards for creating the new data files for these new tables you'll need?

**Comment by gwynnemeeks** [comment]
@luketlancaster  @mtgill I tried splitting them where it made sense to me. Part 2 is #104 More feedback would be great. Also on #91 and #108 which is where I split the add Staff to Ride tickets
